### DISCUSSION TITLE
Why can't GitHub return to the code sharing itself?

### DISCUSSION THREAD

**Discussion by 8u8m** [community/community#120513] [Discover: GitHub Best Practices]
Someone using GitHub to propagate politics in an attempt to split China.Why can't GitHub return to the code sharing itself?

https://github.com/1026576357/wx_qh_shop
https://github.com/cirosantilli/china-dictatorship
https://github.com/gege-circle/home
https://github.com/cirosantilli/china-dictatorhsip-7
https://github.com/tjqJ62cESiHPj6DdR6vXDAcPp/cihna-dictatorshrip-8
https://github.com/cheezcharmer/chinese-dictatorship
https://github.com/panbinibn/OpenPacketFix_
https://github.com/zpc1314521/PCL2
https://github.com/Daravai1234/china-dictatorship
https://github.com/b0LBwZ7r5HOeh6CBMuQIhVu3-s-random-fork/PCL2
https://github.com/lTbgykio/Books-Free-Books
https://github.com/cirosantilli/linux-kernel-module-cheat
https://github.com/Dujltqzv/Some-Many-Books
https://github.com/cirosantilli/x86-bare-metal-examples
https://github.com/cirosantilli/cirosantilli

**Comment by ettaboyle** [comment]
Hi @8u8m  👋🏻 We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our [abuse reporting tools](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam). Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. We are going to close this post, but for this and any future incidents, please refer to the links above.

Thank you!
